pkg/probe/ha-checksum.go

```diff
@@ -0,0 +1,42 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type HAChecksumResults struct {
+	IsManageMaster int    `json:"is_manage_master"`
+	IsRootMaster   int    `json:"is_root_master"`
+	SerialNo       string `json:"serial_no"`
+}
+
+type HAChecksum struct {
+	Results []HAChecksumResults `json:"results"`
+}
+
+func probeSystemHAChecksum(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		IsMaster = prometheus.NewDesc(
+			"fortigate_ha_role",
+			"Master/Slave information",
+			[]string{"name", "serial_no"}, nil,
```

Comment:
```suggestion
			[]string{"role", "serial"}, nil,
```
The other metrics call it `serial`. `name` is a bit ambiguous, I think `role` is better.

Reply:
it make sense